### PR TITLE
python jsbeautifier - handle non-ascii chars

### PR DESCRIPTION
**Pull request by virajkanwade**
# Description
- [x] Source branch in your fork has meaningful name (not `master`)

Fixes Issue: 
Issue #1809

# Before Merge Checklist 
These items can be completed after PR is created.

(Check any items that are not applicable (NA) for this PR)

- [x] JavaScript implementation
- [x] Python implementation (NA if HTML beautifier)
- [x] Added Tests to data file(s)
- [x] Added command-line option(s) (NA if
- [x] README.md documents new feature/option(s)

### CODE REVIEW

**Comment by virajkanwade** [comment]
Please note this is tested only for the specific eval string mentioned in the issue and Python 3.8.3.

**Comment by bitwiseman** [comment]
@virajkanwade  Thanks, I added the equivalent automated tests.

**Comment by virajkanwade** [comment]
Thanks @bitwiseman 
Like I had mentioned I have tested it only for Python 3.8.3. I see the tests have failed for 2.7
https://dev.azure.com/beautifier-io/js-beautify/_build/results?buildId=393&view=logs&j=fd0a0ac8-dbd4-57ad-1a65-f40c557f8d20&t=d6670fde-d031-5f1f-1c0b-3ce9150a00bb&l=447

File: python/jsbeautifier/unpackers/packer.py [hunk 1]
```diff
@@ -61,7 +61,8 @@ def lookup(match):
         word = match.group(0)
         return symtab[unbase(word)] or word
 
-    source = re.sub(r'\b\w+\b', lookup, payload)
+    payload = payload.replace("\\\\", "\\").replace("\\'", "'")
+    source = re.sub(r'\b\w+\b', lookup, payload, flags=re.ASCII)
```
Comment (bitwiseman) [inline python/jsbeautifier/unpackers/packer.py:65]:
```suggestion
    if sys.version_info.major == 2:
        source = re.sub(r'\b\w+\b', lookup, payload)    
    else:
        source = re.sub(r'\b\w+\b', lookup, payload, flags=re.ASCII)
```

**Pull request merged**